Applications/ctkCommandLineModuleExplorer/ctkCmdLineModuleExplorerShowXmlAction.h

```diff
@@ -25,7 +25,14 @@
 
 #include "ctkCmdLineModuleReference.h"
 
-#include <QAction>
+#include <QtGlobal>
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
```

Comment:
Qt4 support was dropped in the CTK based on the work that closed https://github.com/commontk/CTK/issues/1006